### PR TITLE
Feature/13 add logic to settings page

### LINKED ISSUE #13: Add logic to settings page
A user should be able to configure the app in the settings page.

Current state:
The settings page right now only has the ui part implemented.

Target situation:
Add controller and logic to settings page.
For example the User should be able to switch between dark and light mode.

### PR DESCRIPTION
**Pull request by benediktscheffel**

### CODE REVIEW

File: lib/features/settings/pages/settings_page.dart
Comment (leotschritter) [inline lib/features/settings/pages/settings_page.dart:1]: I don't think we need settings in the bottom navigation bar if we are already in settings

File: lib/features/settings/services/persistence/implementation/settings_shared_preferences_persistence_service.dart
Comment (leotschritter) [inline lib/features/settings/services/persistence/implementation/settings_shared_preferences_persistence_service.dart:1]: Do we really want to have german as the fallback language?

Comment (benediktscheffel) [reply lib/features/settings/services/persistence/implementation/settings_shared_preferences_persistence_service.dart:1]: Set the language to en-GB as fallback

File: lib/features/settings/pages/settings_page.dart
Comment (benediktscheffel) [reply lib/features/settings/pages/settings_page.dart:1]: image
THis better?

Comment (benediktscheffel) [reply lib/features/settings/pages/settings_page.dart:1]: I removed the bottom navigation. Now the user can go back to the main screen with the back arrow on the top left.

**Review by leotschritter — APPROVED** [review]

**Pull request merged**